src/os/view/dashboard/index.tsx

```diff
@@ -10,7 +10,7 @@ const Dashboard = () => {
 
   return (
     <Row gutter={[24, 24]} justify="center">
-      <Col span={24} style={{ maxWidth: 1920, width: '100%' }}>
```

Comment:
There're some bugs if we remove width: 100%, please review my old commits to inspect it @thiennv-qng

Reply:
https://github.com/DescartesNetwork/senhub/commit/b835ce2ecdcb86e1b8937e54d50419f1dbf9d751#commitcomment-61881622